templates/bundles/PrestaShopBundle/Admin/Module/Includes/modal_import.html.twig

```diff
@@ -23,68 +23,59 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
 <div id="module-modal-import" class="modal modal-vcenter fade" role="dialog" data-backdrop="static" data-keyboard="false">
-    <div class="modal-dialog">
-        <!-- Modal content-->
-        <div class="modal-content">
-            <div class="modal-header">
-                <h4 class="modal-title module-modal-title">{{ 'Upload a module'|trans({}, 'Admin.Modules.Feature') }}</h4>
-                <button id="module-modal-import-closing-cross" type="button" class="close">&times;</button>
+  <div class="modal-dialog">
+    <!-- Modal content-->
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title module-modal-title">{{ 'Upload a module'|trans({}, 'Admin.Modules.Feature') }}</h4>
+        <button id="module-modal-import-closing-cross" type="button" class="close">&times;</button>
+      </div>
+      <div class="modal-body">
+        {% if level <= constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_UPDATE') %}
+          <div class="alert alert-danger" role="alert">
+            <p class="alert-text">
+              {{ errorMessage }}
+            </p>
+          </div>
+        {% else %}
+          <form action="#" class="dropzone" id="importDropzone">
+            <div class="module-import-start">
+              <i class="module-import-start-icon material-icons">cloud_upload</i><br/>
+              <p class="module-import-start-main-text">
+                {{ 'Drop your module archive here or [1]select file[/1]'|trans({}, 'Admin.Modules.Feature')|replace({'[1]' : '<a href="#" class="module-import-start-select-manual">', '[/1]' : '</a>'})|raw }}
+              </p>
+              <p class="module-import-start-footer-text">
```

Comment:
🤯  what

this was broken before, right?

Reply:
@matks What do you mean?

Reply:
My comment is not in the right place.

The code today looks like this:
```
<p class=module-import-processing-footer-text>
```
Is'nt this broken? Is it OK to have class= without quotes?

The code you wrote is good, I'm surprised by the code that was there before your changes.

Reply:
@matks Oh god, and the single quotes.....
I will search whole templates folder and fix all occurences. 👍

Reply:
Solved ;-)